recipe/meta.yaml

```diff
@@ -37,10 +37,6 @@ test:
     - if not exist "%LIBRARY_LIB%\\yaml.lib" exit 1   # [win]
     - if not exist "%LIBRARY_BIN%\\yaml.dll" exit 1   # [win]
 
-    # Check cmake and pkg-config scripts
-    - test -f "${PREFIX}/cmake/yamlConfig.cmake"      # [not win]
-    - test -f "${PREFIX}/lib/pkgconfig/yaml-0.1.pc"   # [not win]
```

Comment:
Should we keep the `pkgconfig` test?

Reply:
Happy either way - let me know what you prefer.

Reply:
Let's keep the test